lib/tictactoe.rb

```diff
@@ -0,0 +1,85 @@
+require_relative './helper.rb'
+
+class TicTacToe
+  include Helperable
+  attr_accessor :board
+
+  def initialize(board)
+    @board = board
+  end
+
+  def welcome()
+    puts ' '
+    puts ' '
+    puts '|    | |=== |    |===   ===  |\  /| |=== '
+    puts '| /\ | |=== |    |     |   | | \/ | |=== '
+    puts '|/  \| |=== |=== |===   ===  |    | |=== '
+    puts ' '
+    welcome_board
+  end
+
+  def welcome_board()
+    puts ' <--The Tic Tac Toe-->'
+    puts '         | X | O        '
+    puts '      ***********       '
+    puts '       X | O | X        '
+    puts '      ***********       '
+    puts '         | O |          '
+    puts ' <--The Tic Tac Toe-->'
+    puts "\n"
+  end
+
+  def show_board(board = @board)
+    message = (board == @board ? ' Current score listing' : ' Current available positions')
+    puts message
+    puts ' <--The Tic Tac Toe-->'
+    puts "       #{board[0]}  | #{board[1]} | #{board[2]}         "
+    puts '       ***********       '
+    puts "       #{board[3]}  | #{board[4]} | #{board[5]}        "
+    puts '       ***********       '
+    puts "       #{board[6]}  | #{board[7]} | #{board[8]}        "
+    puts ' <--The Tic Tac Toe-->'
```

Comment:
- [ ] No `puts/print` and `get.chomp` outside `main.rb` file. (lines #35 to #41 please refactor the **show_board** method so you can print its result in the main instead of doing it here)

- [ ] Keep your code DRY (don't repeat yourself) lines #35 to #41 are repetitive and can the result can be achieved with fewer lines if a loop is used, let's work on it.